index.js

```diff
@@ -1,5 +1,6 @@
 const { EventEmitter } = require('events');
-const bip39 = require('bip39');
+const { Buffer } = require('buffer');
```

Comment:
Honestly a bit confused why this is necessary... but it seems to be for lint? It's just a node method? cc @Gudahtt

Reply:
What did the lint error say?

Reply:
``` 
103:11  error  'Buffer' is not defined  no-undef
104:11  error  'Buffer' is not defined  no-undef
```

<img width="521" alt="Screen Shot 2022-05-10 at 1 17 31 PM" src="https://user-images.githubusercontent.com/34557516/167686516-f3eb60b4-ab63-43cc-8524-1cfacedaa253.png">

Reply:
In a Node.js context, Buffer is a global. ESLint is complaining here because we've declared the environment as being `commonjs` in the ESLint config, so it is not aware that the Node.js globals exist.

The globals are set correctly in our Node.js ESLint configuration, but that isn't being used by this library for some reason. So that would be another way to solve this. Not a blocker though, this seems OK for now.